transport/session/src/types.rs

```diff
@@ -34,7 +36,88 @@ impl SessionId {
     }
 }
 
+impl Display for SessionId {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}:{}", self.peer, self.tag)
+    }
+}
+
+// Inner MTU size of what the HOPR payload can take (payload - peer_id - application_tag)
+pub const INNER_MTU_SIZE: usize = PAYLOAD_SIZE - 32 - 2;
```

Comment:
Define a constant for `32` at the place where the PeerID is encoded as such.
For `2` there could be `std::mem::size_of::<hopr_internal_types::protocol::Tag>()`